src/types/index.ts

```diff
@@ -157,6 +157,7 @@ export interface IMultisigConsensus {
 
 export interface IMultisigAccountRaw {
   contractId: string;
+  address: string;
```

Comment:
This is a duplicate of `gaAccountId`. Recently I removed this duplicate. Please avoid adding it again as the multisig vaults is a totally different thing than regular accounts.